website/content/en/docs/deployment/cognito-rds-s3/guide.md

```diff
@@ -26,9 +26,15 @@ Refer to the [general prerequisites guide]({{< ref "/docs/deployment/prerequisit
     1. Configure Ingress
 2. Deploy Kubeflow. Choose one of the two options to deploy kubeflow:
     1. **[Option 1]** Install with a single command:
-        ```sh
-        while ! kustomize build deployments/cognito-rds-s3 | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 30; done
-        ```
+        {{< tabpane persistLang=false >}}
+        {{< tab header="Kustomize" lang="toml" >}}
+        make
```

Comment:
Make command missing rest of input? 

Need to export the DEPLOYMENT_OPTION 

`ex DEPLOYMENT_OPTION=cognito-rds-s3

make deploy-kubeflow`